image/templates/helm/stackrox-secured-cluster/values-scanner.yaml.example

```diff
@@ -1,5 +1,5 @@
 # # NOTE:
-# # The Scanner is only available in the secured clusters on the OpenShift Container Platform.
+# # The Scanner is not available in the secured clusters installed via roxctl.
```

Comment:
Is this referring to scanner-slim(v2) being only available on OCP. Did that change?

I think what this comment should say is:
- scanner-slim is only available in the cluster on OCP
- scannerV4 is not available in secured clusters install via roxctl

Reply:
It seems this comes from commit c0cff8fcaf79ba52699144a6ed131380cd31bf5d from @RTann.
Ross, could you comment on this? I am pretty clueless here.